tests/csapi/rooms_members_local_test.go

```diff
@@ -9,11 +9,11 @@ import (
 )
 
 func TestMembersLocal(t *testing.T) {
-	deployment := Deploy(t, b.BlueprintOneToOneRoom)
+	deployment := Deploy(t, b.BlueprintAlice)
 	defer deployment.Destroy(t)
 
 	alice := deployment.Client(t, "hs1", "@alice:hs1")
-	bob := deployment.Client(t, "hs1", "@bob:hs1")
+	bob := deployment.RegisterUser(t, "hs1", "bob", "bobspassword", false)
```

Comment:
Per your comment at https://github.com/matrix-org/complement/pull/563#issuecomment-1345195887,

> `BlueprintOneToOneRoom`, which was already determined in #516 to provide faulty results, as Bob and Alice would already see eachother's presence changes through that pre-existing one-on-one room.


We could add a comment here with this same context since it's important to have a fresh user that doesn't have any connection to Alice already to actually test whether presence changes come through.